pullrequest.rst

```diff
@@ -119,10 +119,30 @@ homepage.
 Submitting
 ----------
 
-Once you are satisfied with your work you will want to commit your
-changes to your branch. In general you can run ``git commit -a`` and
-that will commit everything. You can always run ``git status`` to see
-what changes are outstanding.
+Once you are satisfied with your work, you will need to commit your
+changes to your branch. Assume you want to add a file which you have changed,
+you will need to enter this command::
+
+   git add <file>
+
+After adding all files of changed, you can commit your changes::
+
+   git commit
+
+Inside the commit editor, commit message's title should start with
+``bpo-NNNNN`` and the major change of this commit, for example::
+
+   bpo-29624: Adds purge step and layout test after uploading files.
+
+Two commands, ``git log`` and ``git status``, provide helpful information
+and detail about commits cand changes.
```

Comment:
Typo ~c~and

FWIW `git status` will display the branch before committing. I agree `git diff` is useful prior to commit. `git log` is helpful for reviewing history and whether the cloned fork is up to date.